engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -4129,7 +4129,8 @@ public void mainHelper() throws Exception
 
   public static void main(final String[] args) throws Exception
   {
-    LoggerUtil.addAppenders();
+//    LoggerUtil.setupMDC("client");
```

Comment:
Why is the change necessary? If it is required, do not use comments, delete lines.